Tools/cases_generator/generate_cases.py

```diff
@@ -1584,6 +1573,17 @@ def wrap_macro(self, mac: MacroInstruction):
             self.out.emit(f"DISPATCH();")
 
 
+def prettify_filename(filename: str) -> str:
+    # Make filename more user-friendly and less platform-specific,
+    # it is only used for error reporting at this point.
```

Comment:
That is incorrect, and is not the reason why we go through this much effort. The prettified filename is incorporated in the output (as part of the header explaining that this was a generated file) and must be the same regardless of where it was generated, otherwise you get spurious diffs when e.g. running the tool on Windows.